tests/foreman/cli/test_import.py

```diff
@@ -70,6 +88,43 @@ def test_import_orgs_default(self):
         )
         self.assertEqual(False in [org in orgs for org in imporgs], False)
 
+    def test_import_orgs_manifests(self):
+        """@test: Import all organizations from the default data set
+        (predefined source) and upload manifests for each of them
+
+        @feature: Import Organizations including Manifests
+
+        @assert: 3 Organizations are created with 3 manifests uploaded
+
+        """
+        files = prepare_import_data()[1]
+        csv_records = csv_to_dataset(files['users'])
+        # create number of manifests corresponding to the number of orgs
+        manifest_list = []
+        for org in set([rec['organization'] for rec in csv_records]):
+            for char in [' ', '.', '#']:
+                org = org.replace(char, '_')
+            man_file = manifests.clone()
+            ssh.upload_file(man_file, '/tmp/{}.zip'.format(org))
```

Comment:
Can `mktemp -d` be run on the remote server before uploading files? Doing so should help prevent collisions.